modules/facts/facts.rb

```
@@ -1,7 +1,7 @@
 include ::Proxy::Log
 begin
   require "facter"
-  require 'facts_plugin'
+  require 'facts/facts_plugin'
```

Maybe it's better to record if loading 'facter' was successful, and only require facts_plugin if it was known to be OK, so we don't mask errors in the loading of facts_plugin.